examples/resources/mezmo_open_telemetry_metrics_source/resource.tf

```diff
@@ -0,0 +1,29 @@
+terraform {
```

Comment:
Are you sure this is the right filename.  According to the docs you provided, we should leave off the provider name: 
> NOTE: In the following conventional paths for templates, <data source name>, <resource name>, and <function name> do not include the provider prefix. if the --provider-name argument is set, the provider prefix will be removed from the file names during migration.

...which would mean `examples/resources/open_telemetry_metrics_source/resource.tf`, no?

Reply:
wait, nvm, that's for migrations.  For resources, what you have looks correct.

```
For examples:

NOTE: In the following conventional paths for examples, <data source name> and <resource name> include the provider prefix as well, but the provider prefix is NOT included in<function name>. 
```
Functions are different, which we don't use.